src/components/ProjectCard.js

```diff
@@ -4,81 +4,116 @@ import CardContent from '@material-ui/core/CardContent';
+        <Grid container>
+          <Grid item container xs={12} >
             {/* Project Title, Watch and Star buttons */}
             <Grid container>
-              <Grid item xs={12} sm={8}><a data-cy='project-url' href={props.projectUrl} style={{ color: 'black' }}><span><u>{props.ownerName}/<b>{props.projectName}</b></u></span></a></Grid>
-              <Grid item xs={6} sm={2}><span>{props.watchers} Watchers</span></Grid> {/* TODO: Revise component*/}
-              <Grid item xs={6} sm={2}><span>{props.stargazers} Stargazers</span></Grid> {/* TODO: Revise component*/}
+              {/* Project Logo */}
+              <Grid>
+                <a href={props.organizationUrl}>
+                  <img alt='logo' style={{ display: 'block', margin: 'auto' }} src={props.organizationAvatarUrl} width='72px' />
+                </a>
+              </Grid>
+              <Grid style={{ marginTop: '25px' }}>
+                <a data-cy='project-url' href={props.projectUrl} style={{ color: 'black' }}>
+                  <span>
+                    <u>
+                      {props.ownerName}/<b>{props.projectName}</b>
+                    </u>
+                  </span>
+                </a>
+              </Grid>
+              <Grid item xs={0} />
+              <Grid item xs={12} sm={6} style={{ marginLeft: '10px', marginTop: '25px' }}>
+                <Chip
+                  variant="outlined"
+                  size="small"
+                  avatar={<Avatar alt="Natacha" src="/images/GithubEye.jpg" />}
+                  label={"Watch " + props.watchers}
+                />
+                <Chip
+                  variant="outlined"
+                  size="small"
+                  avatar={<Avatar alt="Natacha" src="/images/GithubStar.jpg" />}
+                  label={"Star " +props.stargazers}
+                />
+              </Grid>
             </Grid>
             {/* Description & Project Link */}
             <Grid container>
-              <Grid item xs={12} sm={12} style={{ paddingTop: '10px' }}><span>{props.projectDescription}</span></Grid>
               <Grid item xs={12} sm={12} style={{ paddingTop: '10px' }}>
-                <span className='project' style={{ verticalAlign: 'middle' }}><img alt="link" src="/images/link.svg" /> <a href={props.homepage}>{props.homepage}</a></span>
+                <span>{props.projectDescription}</span>
+              </Grid>
+              <Grid item xs={12} sm={12} style={{ paddingTop: '10px' }}>
+                <span className='project' style={{ verticalAlign: 'middle' }}>
+                  <img alt='link' src='/images/link.svg' /> <a href={props.homepage}>{props.homepage}</a>
+                </span>
               </Grid>
               {/* Update & Issue Count */}
               <Grid container style={{ paddingTop: '10px' }}>
                 <Grid item xs={12} sm={4}>
-                  {/* TODO: Requires dynamic windows. Hours, Days, Month needs to return Date. */}
```

Comment:
If we implement TODO task we might just want to remove it.

Reply:
According to the figma, this Todo item is not required unless UX team brings it up again.